flexvol/flexvoldriver.go

```diff
@@ -478,6 +495,6 @@ func main() {
 	initConfiguration()
 
 	if err = rootCmd.Execute(); err != nil {
-		genericUnsupported("not supported", "", err.Error())
+		_ = genericUnsupported("not supported", "", err.Error())
```

Comment:
Should we handle this error explicitly? I think probably should at least log it.

Reply:
I'm a bit hesitant to log this since whoever's calling a command that isn't supported might be doing it periodically, and logging it would lead to large log files (similar to https://github.com/projectcalico/pod2daemon/issues/20). Wdyt?

Reply:
...but this is in main() and not one of the other similar calls, so ok :-)

Reply:
...and genericUnsupported() already logs it